### PR TITLE
Fix K8s job status detection

### PR DESCRIPTION
**Pull request by pditommaso**
This PR fixes the detection of job status that can have one or more failed pods and still some active pods.

### CODE REVIEW

**Comment by munishchouhan** [comment]
While testing I found the issue, where if the job is failed wave stuck in checking the status because job.status.completionTime is null and this if remains true:
https://github.com/seqeralabs/wave/blob/007256216ba4f1ed0210786c1ba4261915fd018f/src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy#L208

**Comment by pditommaso** [comment]
Are you saying the the job can complete having  `completionTime == null` ?

**Comment by munishchouhan** [comment]
> Are you saying the the job can complete having `completionTime == null` ?

when job failed, completionTime is still null, which makes this if statement true

**Comment by munishchouhan** [comment]
ok i think we should remove `|| !job.status.completionTime` from here
because completionTime will only have value in case of success 
https://github.com/seqeralabs/wave/blob/007256216ba4f1ed0210786c1ba4261915fd018f/src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy#L208

File: src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy [hunk 1]
```diff
@@ -205,11 +205,14 @@ class K8sServiceImpl implements K8sService {
                 .readNamespacedJob(name, namespace, null)
         if( !job )
             return null
-        if( job.status.succeeded==1 )
+        if( job.status.active || !job.status.completionTime )
```
Comment (munishchouhan) [inline src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy:208]:
```suggestion
        if( job.status.active )
```

**Comment by pditommaso** [comment]
Gotcha!

**Pull request merged**